.formatter.exs

```diff
@@ -1,6 +1,20 @@
+locals_without_parens = [
+  # Query
+  from: 2,
+
+  # Schema
+  field: :*,
```

Comment:
Can we please be explicit about the arities? `:field` should be 1, 2, 3. The others should be 2 and 3. :)

Reply:
No problem.